moveit_core/online_signal_smoothing/include/moveit/online_signal_smoothing/butterworth_filter.h

```diff
@@ -107,14 +107,15 @@ class ButterworthFilterPlugin : public SmoothingBaseClass
    * @param position_vector array of joint position commands
    * @return True if initialization was successful
    */
-  bool doSmoothing(std::vector<double>& position_vector) override;
+  bool doSmoothing(Eigen::VectorXd& positions, Eigen::VectorXd& velocities, Eigen::VectorXd& accelerations) override;
```

Comment:
I think the docstring in this file is still missing the velocities and accelerations